app/views/tag/_tags_popover.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="main_div">
+  <p class="small_heading"> Follow related tags: </p>
+  <%= form_with remote: true, url: '/subscribe/multiple/tag/:tagnames' do |f| %>
+    <% tags.each do |tag| %>
+	  <div>
+		<input type="checkbox" id="<%= tag.id %>" name="tagnames[]" value="<%= tag.name %>" <% if current_user.following(tag.name) %> checked <% end %> ><span class="label <%= label_name %>" style="margin-left: 3px; margin-right: 10px; display: inline-block;"> <%= tag.name %></span>
```

Comment:
Are you getting indentation issues listed by me in this file many times?
I did comment here around 10 times, but I am surprised each time how they vanish each time.
Please fix the indentations.

Reply:
We require 2 space indentations always in each and every file and in each and every line.

Reply:
I have fixed them so many times repeatedly, I don't understand how it's not reflecting there.

Reply:
I have tried to push it again anyways.

Reply:
It's still not reflecting there, you can see my commits of indentation I have made so many of them trying to fix them. Could you please help me understand what could be wrong here?

Reply:
You did not fixed indentation in it. Please check https://github.com/publiclab/plots2/pull/4555/commits/63d326b4f1d83e2438980c859340154dee25079e.

Reply:
Sir I have tried to override it, it's showing me fixed here in my local repo, I have tried to fix it so many times:

![image](https://user-images.githubusercontent.com/40794215/51045995-f34dbf00-15ea-11e9-8a61-a8156c8ba776.png)

Reply:
Please don;t worry too much. Relax and take a deep breath. This is really small issue. You have to fix indentation, save your changes in the editor, git add app/views/tag/_tags_popover.html.erb, git commit -m "indentaion fixed", git push ...
and all done.

Reply:
Yes sir... thank you.. Is it fixed now?

Reply:
yes

Reply:
thanks